### DISCUSSION TITLE
How does DNSCrypt-Proxy poll servers to determine the ones with the lowest latency?

### DISCUSSION THREAD

**Discussion by ghost** [DNSCrypt/dnscrypt-proxy#2588] [Q&A]
How does DNSCrypt-Proxy poll servers to determine latency? The documentation is a bit vague about it. Does DNSCrypt-Proxy poll a specific domain? Is it possible to change it?

**Comment by jedisct1** [comment]
https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Load-Balancing-Options#how-the-servers-are-sorted

**Comment by ghost** [comment]
It doesn't say what server or domains it queries to determine or does it base on queries made by the user to whichever domain they query?

**Comment by ghost** [comment]
> base on queries made by the user to whichever domain they query

Yes.

**Comment by ghost** [comment]
Thank you!
